peakina/readers/csv.py

```diff
@@ -101,7 +101,11 @@ def csv_meta(
             "df_rows": reader_kwargs["nrows"],
         }
 
-    start = 0 + reader_kwargs.get("skiprows", 0)
+    skiprows = reader_kwargs.get("skiprows", 0)
+    if isinstance(skiprows, list):
+        skiprows = len(skiprows)
```

Comment:
je comprend pas comment ça peut marcher, si tu dis que tu veux skip les rows `2, 5, 10`, tu vas te retrouver à faire `skiprows = 3` et commencer à 3 🤔

Reply:
ok je comprend mieux merci 🙏

Reply:
par contre je viens de regarder la video `After` dans la description et on voit que les noms de colonnes ne sont plus bons

Reply:
bah, c'est surtout que c'est le comportement de "pandas" qui fait cela : 
```python
Python 3.11.6 (main, Oct 23 2023, 22:47:21) [GCC 9.4.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.17.2 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pandas as pd

In [2]: pd.read_csv('~/Downloads/test.csv')
Out[2]:
   colum1  colum2   test
0       1      ex     45
1       6      aa     43
2       3   terst     15

In [3]: pd.read_csv('~/Downloads/test.csv', skiprows=[1, 2])
Out[3]:
   colum1  colum2   test
0       3   terst     15

In [4]: pd.read_csv('~/Downloads/test.csv', skiprows=[0, 2])
Out[4]:
   1      ex   45
0  3   terst   15

In [5]:
```

J'avais pour idee de faire un `skiprows = [x + 1 for x in skiprows]`... mais je me suis poser la question si cela n'affectes pas le kwarg lui meme... qu'en penses tu @andreamouraud  ?

Reply:
haha oui du coup il faudrait commencer à 1 👍

Reply:
enfin, c'est ce qui me parait logique, on peut demander à 1 PM pour confirmer mais je vois pas pourquoi on voudrait enlever le nom des colonnes

Reply:
+1

Updated here : https://github.com/toucantoco/peakina/pull/638/commits/27f1316

DEMO : 
https://github.com/ToucanToco/peakina/assets/22576758/e86c49db-4acf-4a56-af3d-ceceaa17b1bb